Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+
+namespace Singleton
+{
+
+    class Singleton
+    {
+        private static Singleton _singleton;
+
+        public Singleton()
+        {
+        }
+
+         public static Singleton getInstance()
+         {
+             return _singleton ?? new Singleton();
```

Comment:
here I think you forgot the equal sign `=`
it should be like this` return _singleton ??= new Singleton();`
that is why your code shows `False` and two different instances.